controllers/kindergarten-controller.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="de-DE">
+<head>
+    <meta charset="UTF-8">
+    <title>Über uns</title>
+</head>
+<style>
+    table {
+        font-family: Arial, sans-serif;
+        border-collapse: collapse;
+        width: 100%;
+    }
+
+    td, th {
+        border: 1px solid #dddddd;
+        text-align: left;
+        padding: 8px;
+    }
+    tr:nth-child(even) {
+        background-color: #dddddd;
+    }
+</style>
+<body>
+    <header>
+        <h1><em>Über uns</em></h1>
+        <p><em>Unser Kindergarten wurde mit neuen Ideen, Konzepten und Technologien gegründet. <br>
+                Wir glauben, dass jedes Kind einzigartig ist und sein Potenzial entfalten kann. <br>
+                Unser Ziel ist es, geniale, liebevolle und wissbegierige Kinder zu erziehen. <br>
+                Wir nutzen moderne Methoden und Technologien, die Kinder zum Lernen, Forschen und täglichen Entdecken motivieren. <br>
+                Bei uns verbringen Kinder nicht einfach nur Zeit - sie wachsen, entwickeln sich und lernen, echte Menschen zu sein: klug, freundlich und selbstbewusst.
+            </em>
+        </p>
+    </header>
+
+    <main>
+        <section>
+            <article>
+
+            </article>
+        </section>
+    </main>
+
+    <table>
+        <tr>
+            <th>Gebeude</th>
+            <th>Spielhalle</th>
+            <th>Unsere Kleinen</th>
+        </tr>
+        <tr>
+            <td><img src="/images/kindergarten_gebeude/Kindergarten-main.jpg" alt="Main"></td>
+            <td><img src="/images/kindern_spielhalle/spiel_raum.jpg" alt="spiel_raum1"></td>
+            <td><img src="/images/kindern/Geburstag.jpeg" alt="Geburstag"></td>
+        </tr>
+        <tr>
+            <td><img src="/images/kindergarten_gebeude/holl.jpg" alt="holl"></td>
+            <td><img src="/images/kindern_spielhalle/spiel_raum2.jpg" alt="spiel_raum2"></td>
+            <td><img src="/images/kindern/essen.jpg" alt=""></td>
+        </tr>
+        <tr>
+            <td><img src="/images/kindergarten_gebeude/unsere_spielhalle.jpg" alt="Spielhalle"></td>
+            <td><img src="/images/kindern_spielhalle/spiel_raum3.jpg" alt="spiel_raum3"></td>
+            <td><img src="/images/kindern/gruppen.jpg" alt="Gruppen"></td>
+        </tr>
+    </table>
```

Comment:
Hier bei der Tabelle fehlt der Head und Body.
```suggestion
    <table>
    <thead>
        <tr>
            <th>Gebeude</th>
            <th>Spielhalle</th>
            <th>Unsere Kleinen</th>
        </tr>
        </thead>
        <tbody>
        <tr>
            <td><img src="/images/kindergarten_gebeude/Kindergarten-main.jpg" alt="Main"></td>
            <td><img src="/images/kindern_spielhalle/spiel_raum.jpg" alt="spiel_raum1"></td>
            <td><img src="/images/kindern/Geburstag.jpeg" alt="Geburstag"></td>
        </tr>
        <tr>
            <td><img src="/images/kindergarten_gebeude/holl.jpg" alt="holl"></td>
            <td><img src="/images/kindern_spielhalle/spiel_raum2.jpg" alt="spiel_raum2"></td>
            <td><img src="/images/kindern/essen.jpg" alt=""></td>
        </tr>
        <tr>
            <td><img src="/images/kindergarten_gebeude/unsere_spielhalle.jpg" alt="Spielhalle"></td>
            <td><img src="/images/kindern_spielhalle/spiel_raum3.jpg" alt="spiel_raum3"></td>
            <td><img src="/images/kindern/gruppen.jpg" alt="Gruppen"></td>
        </tr>
        </tbody>
    </table>
```